<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CScopeForm</class>
 <widget class="QDialog" name="CScopeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Scope">
      <attribute name="title">
       <string>Scope</string>
      </attribute>
      <widget class="CScopeControl" name="ScopeControl" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>371</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Spectrum">
      <attribute name="title">
       <string>Spectrum</string>
      </attribute>
      <widget class="CSpectrumControl" name="SpectrumControl" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>371</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CScopeControl</class>
   <extends>QWidget</extends>
   <header>cscopecontrol.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CSpectrumControl</class>
   <extends>QWidget</extends>
   <header>cspectrumcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
